MyBlog.API/Controllers/UserController.cs

```diff
@@ -0,0 +1,100 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using MyBlog.Infrastructure.Entities;
+using MyBlog.Service.Areas.Users;
+
+namespace MyBlog.Controllers;
+
+[ApiController]
+[Route("[controller]")]
+public class UserTestController : ControllerBase
+{
+    private readonly ILogger<UserTestController> _logger;
+    private readonly IUserService _userService;
+    
+    public UserTestController(ILogger<UserTestController> logger, IUserService userService)
+    {
+        _logger = logger;
+        _userService = userService;
+    }
+
+    /// <summary>
+    /// Get list of Users
+    /// </summary>
+    /// <returns>Returns User list</returns>
+    /// <response code="200">Success</response>
+    [HttpGet]
+    [ProducesResponseType(StatusCodes.Status200OK)]
+    public async Task<IActionResult> GetListAsync()
+    {
+        var users = await _userService.GetListAsync();
+        
+        return Ok(users);
+    }
+
+    /// <summary>
+    /// Get User by id
+    /// </summary>
+    /// <param name="id">User id</param>
+    /// <returns>Returns User</returns>
+    /// <response code="200">Success</response>
+    /// <response code="404">UserNotFound</response>
+    [HttpGet( "{id:int}")]
+    [ProducesResponseType(StatusCodes.Status200OK)]
+    [ProducesResponseType(StatusCodes.Status404NotFound)]
+    public async Task<IActionResult> GetByIdAsync(int id)
+    {
+        var user = await _userService.GetByIdAsync(id);
+    
+        return Ok(user);
+    }
+    
+    /// <summary>
+    /// Creates a User.
+    /// </summary>
+    /// <param name="user">User object</param>
+    /// <returns>Id created user</returns>
+    /// <response code="200">Success</response>
+    [HttpPost]
+    [ProducesResponseType(StatusCodes.Status200OK)]
+    public async Task<IActionResult> CreateAsync([FromBody] User user)
+    {
+        var idUser = await _userService.CreateAsync(user);
+        
+        return Ok(idUser);
+    }
```

Comment:
in the case with create and update, as a user i wanna see, that my changes is apply, i wanna see new entity, but you return me userId. What i should to do with that, and how can i detect, that my changes apply?)